### PR TITLE
[Fugu API Showcase] Add Electron.js label

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Electron.js label 

Screenshot 2023-02-10 at 16 48 12

### CODE REVIEW

File: site/_includes/partials/showcase-card.njk [hunk 1]
```diff
@@ -32,6 +32,9 @@
         {{ 'i18n.common.launch_app' | i18n(locale) }}
```
Comment (matthiasrohmer) [inline site/_includes/partials/showcase-card.njk:32]: I wonder if the CTA should also change to "Download app" in this case...? WDYT?

Comment (tomayac) [reply site/_includes/partials/showcase-card.njk:32]: Good point. Will make it so.

**Comment by tomayac** [comment]
"Download app" when Electron:

Screenshot 2023-02-13 at 12 49 56

**Review by matthiasrohmer — APPROVED** [review]
Thanks, Tom 🙂

**Pull request merged**